.github/workflows/odroid_arm64.yml

```diff
@@ -0,0 +1,47 @@
+name: odroid_arm64
+
+on:
+  push:
+  pull_request:
+  repository_dispatch:
+    types: [backend_automation]
+  workflow_dispatch:
+
+env:
+  CI_MAKE: make -f .travis.mk
+
+jobs:
+  odroid_arm64:
+    # We want to run on external PRs, but not on our own internal PRs
+    # as they'll be run by the push to the branch.
+    if: ( github.event_name == 'push' ||
+        github.event.pull_request.head.repo.full_name != github.repository ) &&
+        ! endsWith(github.ref, '-notest')
+
+    runs-on: odroid_arm64
+
+    strategy:
+      fail-fast: false
+
+    steps:
+      - uses: actions/checkout@v1
+      - uses: ./.github/actions/environment
+      - name: test
+        env:
+          CMAKE_BUILD_TYPE: Release
```

Comment:
Nit: May be it is better to use `RelWithDebInfo` here?
If Tarantool will crash during tests we can collect coredump with debug info.

Reply:
It's not better, but rather obligatory: we never build Release w/o debuginfo.

Reply:
Ok, sure corrected.